include/container/sycl_iterator.h

```diff
@@ -35,25 +35,25 @@ class BufferIterator {
  public:
   using scalar_t = element_t;
   template <int dim = 1>
-  using buffer_t = cl::sycl::buffer<scalar_t, dim>;
-  using access_mode_t = cl::sycl::access::mode;
-  template <cl::sycl::access::mode acc_md_t =
-                cl::sycl::access::mode::read_write,
-            cl::sycl::access::target access_t =
```

Comment:
`access::target` is now just `target` I think.

Reply:
I converted all `access::target` and to simply `target` in 0d920f0 and this commit also substitute the deprecated `target::global_buffer` with `target::device`